src/SectionsShared/MultiFrame.cs

```diff
@@ -284,7 +284,11 @@ public async Task RemoveFrame(string name)
 				switch (transitionInfo)
 				{
 					case DelegatingFrameSectionsTransitionInfo frameTransition:
+
 						await frameTransition.Run(frameToHide: previousFrame.Frame, frameToShow: nextFrame.Frame, frameToShowIsAboveFrameToHide: true);
+
+						// Collapse the previous frame under the modal to allow the screen reader to ignore the previous frame.
+						previousFrame.Frame.Visibility = Visibility.Collapsed;
```

Comment:
@takla21 Are there cases where we would want to still see the previous frame through the modal? Like with transparency.

@doum-vigeant-nventive Is there a way to mark a view invisible to the eye of the screen reader, but visible to the eye of the user?

Reply:
@jeanplevesque yes there is but it would involve to recursively do the same thing to every UIElement within it

Reply:
> @jeanplevesque good question, it could probably be a problem if we decide to open/close modal using the fadein/fadeout transitions, especially in this context #26. We may want to test first though in case I'm wrong

I think it would be fine because it's aligned with the sections transitions (which are using fade transitions by default).